network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/audio_block.html

```diff
@@ -0,0 +1,28 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags %}
+
+{% block block_content %}
+
+<div class="player-and-caption my-default">
+    <div id="audio-player">
+        <audio class="embedded-audio">
+            <source src={{self.audio.url}} type="audio/mp3" preload="metadata" loop>
```

Comment:
```suggestion
            <source src="{{self.audio.url}}" type="audio/mp3" preload="metadata" loop>
```

Just in case the browser determines %20 as a space and breaks this into separate attributes.